src/Endpoints/class-related-api-proxy.php

```diff
@@ -36,20 +36,19 @@ final class Related_API_Proxy {
 	private $proxy;
 
 	/**
-	 * Used to inject dependencies.
+	 * Constructor.
 	 *
 	 * @param Parsely $parsely Instance of Parsely class.
-	 * @param Proxy   $proxy   Proxy object which does the actual calls to the Parse.ly API.
+	 * @param Proxy   $proxy   Proxy object which does the actual calls to the
+	 *                         Parse.ly API.
```

Comment:
Is there any line character limitation that we're following for this line breaks?

Reply:
[The standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/) state:

> DocBlock text should wrap to the next line after 80 characters of text. If the DocBlock itself is indented on the left 20 character positions, the wrap could occur at position 100, but should not extend beyond a total of 120 characters wide.

To conclude:

- I go for 80 chars anywhere possible.
- In some special cases I've gone for 100 and in one one case for 120, as the comments would take a lot of lines.
- In general, when we have long variable names and types (for example arrays), it's OK to go with 100 chars if this will save us an extra line.
- 120 chars is a last resort when having comments that will just take up too many lines.
- I also look at the code which is close to the comment. Having short lines of code with a really long comment doesn't look coherent.

Let me know if this answers the question.

Reply:
That makes sense. Again, I personally don't like this rule (given we're in the age of ultra hdpi and wide displays), but if that's the standard, happy to adhere to it.